### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by abeltsew**
Created a Desktop Design for All sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,18 +15,28 @@
...
+          <li class="desktopLink">Contact</li>
+          <li class="desktopLink">
+            <img class="mail" src="/images/Mail.svg" alt="Mail" />
+          </li>
+        </ul>
       </nav>
     </header>
 
     <!-- Headline -->
     <section class="headline">
```
Comment (ABDULALI3468) [inline index.html:30]: I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

image

## Figma Design:-

image

File: index.html [hunk 2]
```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
...
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="tools">
-          <li><button class="toolsBtn">CSS</button></li>
-          <li><button class="toolsBtn">HTML</button></li>
-          <li><button class="toolsBtn">Bootstrap</button></li>
-          <li><button class="toolsBtn">Ruby</button></li>
-        </ul>
-        <button class="mainSeeBtn">See Project</button>
-      </div>
```
Comment (ABDULALI3468) [inline index.html]: I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

image

## Figma Design:-

image


## Remember:-

You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.

File: index.html [hunk 3]
```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
...
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="cardBtns">
-            <li><button class="cardBtn">HTML</button></li>
-            <li><button class="cardBtn">BootStrap</button></li>
-            <li><button class="cardBtn">Ruby</button></li>
-          </ul>
+      <div class="cards">
```
Comment (ABDULALI3468) [inline index.html:99]: **1-** Currently, you have **5 cards** but the design has **6 cards** please consider adding one to the project.

**2-** Moreover, in the template, there exist 3 cards in a single row. But in your case in depends upon the width of the screen so on smaller screens it works well but when we go to bigger screens there can be 4 and 5 cards on a  single row. Please consider minimizing the number of cards per row to be 3 in accordance with the design.

## Your Design:-

image


## Figma Design:-

image

File: index.html [hunk 4]
```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
...
+              <li><button class="cardBtn">BootStrap</button></li>
+              <li><button class="cardBtn">Ruby</button></li>
+            </ul>
+          </div>
+          <button class="cardSeeProj">See Project</button>
         </div>
-        <button class="cardSeeProj">See Project</button>
       </div>
     </section>
     <section class="about">
```
Comment (ABDULALI3468) [inline index.html:198]: I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

image


## Figma Design:-

image

## Remember:-
You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.

File: style.css [hunk 5]
```diff
@@ -480,3 +475,168 @@ footer {
...
+  }
+
+  .contactBtn {
+    align-self: flex-start;
+  }
+
+  .contactForm {
+    gap: 0;
+  }
+}
```
Comment (ABDULALI3468) [inline style.css]: **1-** After both the `input tags` you have used some images as separators. They are networking please consider reviewing them! to make it in accordance with the design.

**2-** I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

image

## Figma Design:-

image

## Remember:-
You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.

File: index.html [hunk 6]
```diff
@@ -15,18 +15,28 @@
...
         <img class="mobileMenu" src="images\IconMenu.png" alt="Menu" />
+        <ul class="desktopMenu">
+          <li class="desktopLink">Portfolio</li>
+          <li class="desktopLink">About</li>
+          <li class="desktopLink">Contact</li>
+          <li class="desktopLink">
+            <img class="mail" src="/images/Mail.svg" alt="Mail" />
+          </li>
+        </ul>
       </nav>
```
Comment (ABDULALI3468) [inline index.html:26]: Please consider removing the property of `position: fixed;` from your nav as according to the design it does not to be fixed.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @abeltsew , 



Indeed you have done an exceptional job but still, there are some issues that need to be addressed to complete this project.

## Remember:-

You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font sizes, margins, padding, width, and height are actually similar to the design.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @abeltsew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**